Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
I wrote inverted the "right" way in the code snippet before, so it would be like:
```php
    /**
     * Return the list of colors.
     *
     * @return array
     */
    public static function getAll($isInverted = false)
    {
        $colors = array(
            'aliceblue' => self::ALICEBLUE,
            'antiquewhite' => self::ANTIQUEWHITE,
            //...
        );

        // NEXT_MAJOR: Remove this "if" (when requirement of Symfony is >= 2.8)
        if($isInverted) {
            @trigger_error(
                'The $isInverted argument of '.__METHOD__.' is deprecated since 3.3, to be removed in 4.0.',
                E_USER_DEPRECATED
            );
            $colors[self::FUCHSIA] = 'fuchsia';
            $colors[self::CYAN] = 'cyan';
            return array_flip($colors);
        }
        
        return $colors;
    }
```
After I'll take a look to the tests.